packages/SwingSet/src/devices/bridge-src.js

```diff
@@ -15,9 +17,7 @@ export function buildRootDeviceNode(tools) {
   let { inboundHandler } = getDeviceState() || {};
 
   function inboundCallback(...args) {
-    if (!inboundHandler) {
-      throw new Error(`inboundHandler not yet registered`);
-    }
+    assert(inboundHandler, d`inboundHandler not yet registered`);
```

Comment:
There's no `throw` here. Is there a reason for this one to be different?

Reply:
If the `inboundHandler` is truthy, this is supposed to continue normally, not throw. Just as the original `if` statement would.